conf/themes/silverstripe/class.twig

```diff
@@ -80,6 +80,7 @@
 {% endblock %}
 
 {% block class_signature -%}
+    {% if class.final %}final {% endif %}
```

Comment:
I did see your change, but please always check that this is not an upstream bug ;)

Fixed as: https://github.com/code-lts/doctum/commit/4fae085c573a3d75a0a0d059b125620dad506a50
Confirmed on https://develdocs.phpmyadmin.net/phpmyadmin/PhpMyAdmin/DbTableExists.html